pkg/controller/extendeddaemonsetreplicaset/filters.go

```diff
@@ -18,6 +19,14 @@ import (
 	podutils "github.com/datadog/extendeddaemonset/pkg/controller/utils/pod"
 )
 
+var (
+	ignoreEvictedPods = false
+)
+
+func init() {
+	pflag.BoolVarP(&ignoreEvictedPods, "ignoreEvictedPods", "i", ignoreEvictedPods, "Enabling this will force creating new pods on nodes where pods are evicted")
```

Comment:
```suggestion
	pflag.BoolVarP(&ignoreEvictedPods, "ignoreEvictedPods", "i", ignoreEvictedPods, "Enabling this will force new pods creation on nodes where pods are evicted")
```